addon/adapter.go

```diff
@@ -5,10 +5,11 @@ Tackle hub/addon integration.
 package addon
 
 import (
+	"os"
+
 	"github.com/konveyor/controller/pkg/logging"
 	"github.com/konveyor/tackle2-hub/settings"
 	"golang.org/x/sys/unix"
-	"os"
```

Comment:
We need to decide on import ordering.  Looks like your IDE may be organizing imports using _goimports_.  revert this change so we don't have the thrashing.

Reply:
Thanks, I missed this unwanted change, reverting.

On the formatting tool, happy to discuss it, for me personally, I prefer "separated" imports (from goimports), but e.g. not removing empty comment lines like goimports do by default (and that's what we use quite often in go files), but I can accept whatever we agree on.